os_migrate/plugins/modules/import_workload_transfer_volumes.py

```diff
@@ -557,9 +563,10 @@ def _detach_destination_volumes(self):
 
 def run_module():
     argument_spec = openstack_full_argument_spec(
+        auth=dict(type='dict', no_log=True, required=True),
         data=dict(type='dict', required=True),
         conversion_host=dict(type='dict', required=True),
-        ssh_key_path=dict(type='str', required=True),
+        ssh_key_path=dict(type='str', required=True, no_log=True),
```

Comment:
Ditto

Reply:
Now reverted in https://github.com/os-migrate/os-migrate/pull/484/commits/dcb6ba052d85e887f34570d66b489a6b7a0a09da